composer.json

```diff
@@ -16,6 +16,7 @@
         }
     ],
     "minimum-stability": "RC",
+    "prefer-stable": true,
```

Comment:
PR looks good to me, but I'm unsure about this part. If we have `minimum-stability: RC`, will that mean when we publish 1.1@RC it will automatically update, or does prefer-stable _always_ hold that down?

Wondering if instead we should tell people to opt-in by adding these lines if needed.

Reply:
Yeah reading further I think the better advice will be to opt in to RCs by doing something like:

```json
{
  "require": {
    "altis/altis": "^1.0.0",
    "altis/cms": "@RC"
  }
}
```

That will opt in to `1.x.x@RC` for that package, only thing to be aware of is if the opted in package has dependencies that are also RC - those would fail unless also specified but I don't think that's going to happen in our case.